packages/manager/modules/pci/src/projects/creating/controller.js

```diff
@@ -64,8 +66,7 @@ export default class PciProjectCreatingCtrl {
 
   startOrderFollowUpPolling(interval = ORDER_FOLLOW_UP_POLLING_INTERVAL) {
     this.orderFollowUpPolling = this.$timeout(() => {
-      this.pciProjectCreating
-        .getOrderFollowUp(this.orderId)
+      this.PciProjectsService.getOrderFollowUp(this.orderId)
```

Comment:
Coul it make sense to reuse stuff from the `ng-ovh-order-tracking` module

Reply:
I am note sure it help.
The only things that i need is to reuse the getOrderFollowUp which is already implemented.